pkg/api/handlers/generic/containers_stats.go

```diff
@@ -56,7 +57,7 @@ func StatsContainer(w http.ResponseWriter, r *http.Request) {
 
 	stats, err := ctnr.GetContainerStats(&libpod.ContainerStats{})
 	if err != nil {
-		InternalServerError(w, errors.Wrapf(err, "Failed to obtain Container %s stats", name))
+		handlers.InternalServerError(w, errors.Wrapf(err, "Failed to obtain Container %s stats", name))
```

Comment:
No capitalized errors.

Reply:
to be consistent with the other api, we must use caps.